erpnext/stock/report/product_bundle_balance/product_bundle_balance.py

```diff
@@ -0,0 +1,187 @@
+
+			data.append(parent_row)
+			data += child_rows
+
+	return columns, data
+
+
+def get_columns():
+	columns = [
+		{"fieldname": "item_code", "label": _("Item"), "fieldtype": "Link", "options": "Item", "width": 300},
+		{"fieldname": "warehouse", "label": _("Warehouse"), "fieldtype": "Link", "options": "Warehouse", "width": 100},
+		{"fieldname": "uom", "label": _("UOM"), "fieldtype": "Link", "options": "UOM", "width": 70},
+		{"fieldname": "bundle_qty", "label": _("Bundle Qty"), "fieldtype": "Float", "width": 100},
+		{"fieldname": "actual_qty", "label": _("Actual Qty"), "fieldtype": "Float", "width": 100},
+		{"fieldname": "minimum_qty", "label": _("Minimum Qty"), "fieldtype": "Float", "width": 100},
+		{"fieldname": "item_group", "label": _("Item Group"), "fieldtype": "Link", "options": "Item Group", "width": 100},
+		{"fieldname": "brand", "label": _("Brand"), "fieldtype": "Link", "options": "Brand", "width": 100},
+		{"fieldname": "description", "label": _("Description"), "width": 140},
+		{"fieldname": "company", "label": _("Company"), "fieldtype": "Link", "options": "Company", "width": 100}
+	]
+	return columns
+
+
+def get_items(filters):
+	pb_details = frappe._dict()
+	item_details = frappe._dict()
+
+	conditions = get_parent_item_conditions(filters)
+	parent_item_details = frappe.db.sql("""
+		select item.name as item_code, item.item_name, pb.description, item.item_group, item.brand, item.stock_uom
+		from `tabItem` item
+		inner join `tabProduct Bundle` pb on pb.new_item_code = item.name
+		where ifnull(item.disabled, 0) = 0 {0}
+	""".format(conditions), filters, as_dict=1)  # nosec
+
+	parent_items = []
+	for d in parent_item_details:
+		parent_items.append(d.item_code)
+		item_details[d.item_code] = d
+
+	if parent_items:
+		child_item_details = frappe.db.sql("""
+			select
+				pb.new_item_code as parent_item, pbi.item_code, item.item_name, pbi.description, item.item_group, item.brand,
+				item.stock_uom, pbi.uom, pbi.qty
+			from `tabProduct Bundle Item` pbi
+			inner join `tabProduct Bundle` pb on pb.name = pbi.parent
+			inner join `tabItem` item on item.name = pbi.item_code
+			where pb.new_item_code in ({0})
+		""".format(", ".join(["%s"] * len(parent_items))), parent_items, as_dict=1)  # nosec
+	else:
+		child_item_details = []
+
+	child_items = set()
+	for d in child_item_details:
+		if d.item_code != d.parent_item:
+			pb_details.setdefault(d.parent_item, []).append(d)
+			child_items.add(d.item_code)
+			item_details[d.item_code] = d
+
+	child_items = list(child_items)
+	return item_details, pb_details, parent_items, child_items
+
+
+def get_stock_balance(filters, items):
+	sle = get_stock_ledger_entries(filters, items)
+	stock_balance = frappe._dict()
+	for d in sle:
+		stock_balance.setdefault(d.item_code, frappe._dict())[d.warehouse] = d
+	return stock_balance
+
+
+def get_stock_ledger_entries(filters, items):
+	if not items:
+		return []
+
+	item_conditions_sql = ' and sle.item_code in ({})' \
```

Comment:
Also the `item_conditions_sql` is not getting formatted properly
![Screenshot 2019-05-29 at 7 16 07 PM](https://user-images.githubusercontent.com/42651287/58562366-bbb47c00-8246-11e9-816f-91b5ea61c78a.png)
```suggestion
	item_conditions_sql = ' and sle.item_code in (%s)' \
```